packages/core/util/jexlStrings.ts

```diff
@@ -14,21 +14,10 @@ const compilationCache: Record<string, any> = {}
  */
 export function stringToJexlExpression(
   str: string,
-  options: {
-    /**
-     * if passed, the compiled function will check at runtime that the proper
-     * number of arguments were passed to it
-     */
-    verifyContextVariable?: string[]
-  } = {},
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   jexl?: any,
 ) {
-  const { verifyContextVariable } = options
-
-  const cacheKey = `${
-    verifyContextVariable ? verifyContextVariable.join(',') : 'nosig'
-  }|${str}`
+  const cacheKey = `${'nosig'}|${str}`
   if (!compilationCache[cacheKey]) {
     const match = str.startsWith('jexl:')
     if (!match) {
```

Comment:
removed verifyContextVariable here. Let me know if it should be added back or if there is another way we should verify the jexlStrings when option is passsed